### ISSUE TITLE
[event flow] Error in rendering the chart

### ISSUE THREAD

**Issue by kamalkeshavani-aiinside** [apache/superset#13544]
When moving the cursor to chart area, it results in render error.

### Expected results

User can move cursor to chart area.

### Actual results

The chart is loaded when query is executed, but results in render error when cursor moves to chart area.

#### Screenshots

Loaded chart:
![image](https://user-images.githubusercontent.com/74634977/110594721-85988000-81c0-11eb-8386-c758ab905d9b.png)

Error:
![image](https://user-images.githubusercontent.com/74634977/110594674-7580a080-81c0-11eb-84b0-cfb1295fd578.png)

#### How to reproduce the bug

1. Create an Event Flow chart from 'flights' table from examples.
2. Run the query
3. Move the cursor to chart area.

Error in UI:
```  
    in i
    in div
    in i
    in div
    in o
    in withBoundingRects()
    in div
    in t
    in div
    in div
    in t
    in div
    in t
    in div
    in t
    in div
    in t
    in div
    in t
    in div
    in t
    in h
    in n
    in div
    in w
    in A
    in N
    in div
    in div
    in Styled(div)
    in d
    in ee
    in x
    in div
    in div
    in t
    in div
    in Styled(div)
    in Er
    in div
    in div
    in Styled(div)
    in ho
    in x
    in T
    in m
    in Unknown
    in s
    in yo
```
Console Logs:
```
172.17.0.1 - - [10/Mar/2021:07:53:16 +0000] "GET /api/v1/chart/0 HTTP/1.1" 404 24 "http://localhost:8080/superset/explore/?form_data=%7B%22viz_type%22%3A%22event_flow%22%2C%22datasource%22%3A%2210__table%22%2C%22url_params%22%3A%7B%7D%2C%22time_range_endpoints%22%3A%5B%22inclusive%22%2C%22exclusive%22%5D%2C%22granularity_sqla%22%3A%22ds%22%2C%22time_range%22%3A%22No+filter%22%2C%22entity%22%3A%22FLIGHT_NUMBER%22%2C%22all_columns_x%22%3A%22AIRLINE_DELAY%22%2C%22row_limit%22%3A10000%2C%22order_by_entity%22%3Atrue%2C%22min_leaf_node_event_count%22%3A1%2C%22adhoc_filters%22%3A%5B%5D%2C%22all_columns%22%3A%5B%5D%2C%22extra_form_data%22%3A%7B%22custom_form_data%22%3A%7B%7D%2C%22override_form_data%22%3A%7B%7D%2C%22append_form_data%22%3A%7B%7D%7D%7D" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.192 Safari/537.36"

172.17.0.1 - - [10/Mar/2021:07:53:16 +0000] "POST /superset/log/?explode=events HTTP/1.1" 200 1 "http://localhost:8080/superset/explore/?form_data=%7B%22viz_type%22%3A%22event_flow%22%2C%22datasource%22%3A%2210__table%22%2C%22url_params%22%3A%7B%7D%2C%22time_range_endpoints%22%3A%5B%22inclusive%22%2C%22exclusive%22%5D%2C%22granularity_sqla%22%3A%22ds%22%2C%22time_range%22%3A%22No+filter%22%2C%22entity%22%3A%22FLIGHT_NUMBER%22%2C%22all_columns_x%22%3A%22AIRLINE_DELAY%22%2C%22row_limit%22%3A10000%2C%22order_by_entity%22%3Atrue%2C%22min_leaf_node_event_count%22%3A1%2C%22adhoc_filters%22%3A%5B%5D%2C%22all_columns%22%3A%5B%5D%2C%22extra_form_data%22%3A%7B%22custom_form_data%22%3A%7B%7D%2C%22override_form_data%22%3A%7B%7D%2C%22append_form_data%22%3A%7B%7D%7D%7D" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.192 Safari/537.36"
```

### Environment

(please complete the following information):

- superset version: `v1.0.1`
- python version: `v3.7.9`
Using official docker image without any changes to config.

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

Also can reproduce the error in master.

**Comment by junlincc** [comment]
@kamalkeshavani-aiinside thanks for reporting. I also saw this error in latest master couple days ago.

**Comment by daniel10012** [comment]
@junlincc the SQL query generated seems wrong and has to be fixed in the event flow chart

![image](https://user-images.githubusercontent.com/10360991/110849402-29476600-827d-11eb-9364-f20e697531b4.png)

![image](https://user-images.githubusercontent.com/10360991/110849600-38c6af00-827d-11eb-9d08-9e6f495185d7.png)

**Comment by junlincc** [comment]
@daniel10012 thanks for confirming. unfortunately we will have to wait for a couple of sprints.....

**Comment by kamalkeshavani-aiinside** [comment]
I can see that issue is fixed now in 1.2rc
